drive_to_ball.py

```diff
@@ -10,11 +12,14 @@ def __init__(self):
         self.get_ball_coordinates = GetCoordinates("ball")
         self.driver = DriveTowards()
         self.to_goal = TurnToGoal()
+        self.game_status = GameStatus()
```

Comment:
It seems to me that the drive_towards class only reads the game status on initialization, which makes the loop still endless.